tests/signedUrlIntegrationTest.js

```diff
@@ -1,10 +1,16 @@
-describe('Signed urls', function () {
+fdescribe('Signed urls', function () {
   var OwnCloud = require('../src/owncloud')
   var config = require('./config/config.json')
 
   // LIBRARY INSTANCE
   var oc
 
+  // PACT setup
+  const Pact = require('@pact-foundation/pact-web')
+  const provider = new Pact.PactWeb()
+  const { setGeneralInteractions, ocsMeta } = require('./pactHelper.js')
+  const { accessControlAllowHeaders, accessControlAllowMethods, validAuthHeaders, origin } = require('./pactHelper.js')
```

Comment:
`
```suggestion
  const {
    setGeneralInteractions,
    ocsMeta,
    accessControlAllowHeaders,
    accessControlAllowMethods,
    validAuthHeaders,
    origin
} = require('./pactHelper.js')
```
requiring from a same file

Reply:
that creates more lines of code